cmd/utils.go

```diff
@@ -225,10 +238,13 @@ func getProjectConfig() ProjectConfig {
 	}
 	return *projectConfig
 }
-func getProjectName() string {
-	projectDir := getProjectDir()
+func getProjectName() (string, error) {
+	projectDir, err := getProjectDir()
+	if err != nil {
+		return "", err
```

Comment:
I think this should still return the project name if it can, which means we can continue this function and return `err` at the end.